Core/AppRuntime/Source/AppRuntime_Android.cpp

```diff
@@ -14,6 +14,16 @@ namespace Babylon
     {
         std::stringstream ss{};
         ss << "Uncaught Error: " << error.what() << std::endl;
+
+        try
+        {
+            throw;
+        }
+        catch (const Napi::Error& error)
+        {
+            ss << GetErrorInfos(error) << std::endl;
+        }
```

Comment:
Seems like this is all the same between platforms. Factor out into a shared function?

Reply:
makes sense. fixed.